cmd/check-imports/main.go

```diff
@@ -2,32 +2,60 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Command check-imports inspects a source tree for blacklisted imports.
+//
+// Example:
+//
+//  $> check-imports -b="github.com/gonum/.*,math/rand"
+//  $> check-imports -b="github.com/gonum/.*,math/rand" .
+//  $> check-imports -b="github.com/gonum/.*,math/rand" /some/dir /other/dir
 package main
 
 import (
+	"flag"
 	"log"
 	"os"
+	"strings"
 
 	"gonum.org/v1/tools/internal/imports"
 )
 
-var blacklist = []string{
-	"github.com/gonum/.*", // prefer gonum.org/v1/gonum
-	"math/rand",           // prefer golang.org/x/exp/rand
-}
+var defaultBlacklist = strings.Join(
```

Comment:
I'm not sure we should do this if we're making a public utility - the cost of adding the lines to our invocation is not huge.

Reply:
done.